pkg/capabilities/consensus/ocr3/types/ocr3_config_types.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+option go_package = "capabilities/consensus/ocr3/types";
+
+package ocr3_config_types;
+
+message ReportingPluginConfig {
+  uint32 maxQueryLengthBytes = 1;
+  uint32 maxObservationLengthBytes = 2;
+  uint32 maxReportLengthBytes = 3;
+  uint32 maxOutcomeLengthBytes = 4;
+  uint32 maxReportCount = 5;
+  uint32 maxBatchSize = 6;
+  uint64 outcomePruningThreshold = 7;
+  bool uniqueReports = 8;
```

Comment:
You can remove uniqueReports. It's hardcoded to true in the contract so it also needs to stay =true in the node.